Client/src/Forms/useForm.js

```diff
@@ -1,11 +1,17 @@
+/** [useForm.js]
+* Summary. 
+This useForm is for the sign up page, it checks if the values entered in the form are correct, then handles the submit.
```

Comment:
Swap contents of Summary and Description.

Reply:
done